src/screens/FeedScreen/Header.js

```diff
@@ -35,12 +35,12 @@ import StringConstant from '../../utils/string/StringConstant';
 import useFeedHeader from './hooks/useFeedHeader';
 import {DEFAULT_PROFILE_PIC_PATH, PRIVACY_PUBLIC} from '../../utils/constants';
 import {calculateTime} from '../../utils/time';
-import {colors} from '../../utils/colors';
 import {fonts, normalizeFontSize} from '../../utils/fonts';
 import ShareAndroidIcon from '../../assets/icons/images/share-for-android.svg';
 import TrashRed from '../../assets/icons/images/trash-red.svg';
 import BottomSheetMenu from '../../components/BottomSheet/BottomSheetMenu';
 import ShareUtils from '../../utils/share';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `colors` has been updated to `COLORS`. This change is consistent and improves the code readability by making it clear that `COLORS` is a constant.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->